proposals/Fleet-Windows-OS-vulns.md

```diff
@@ -0,0 +1,139 @@
+# Detecting Windows OS vulnerabilities
+
+The first step in detecting Windows vulnerabilities is understanding how they are remediated. Using
+the CISA list of know vulnerabilities as a sample, it was determined that all vulnerability
+remediations follow the form of *applying some software patch* and then *following some
+steps (if any)* so it follows that to determine if a system is susceptible to a vulnerability we will need to
+check whether a specific software patch was applied and also if the provided steps were followed. So
+basically we have three problems we need to solve: 
+1. We need a list of vulnerabilities + remediations. Each remediation is composed of a patch + some steps.
+2. For a given vulnerability we need to determine whether the proper patch was applied.
```

Comment:
this is not always true, right? there are nuances depending on the remediation type model. Eg: a Workaround might not have a patch.

Reply:
Yeah, but we only care about vulnerabilities for which there are patches (because that's the only thing we can accurately detect). And just to clarify, a vulnerability can have both a patch and a workaround, if the user applies either the vulnerability can be remediated.

Reply:
that makes sense, thanks 👍